### ISSUE TITLE
Update cannon.js to cannon-es

### ISSUE THREAD

**Issue by kikohs** [cables-gl/cables_docs#61]
The Cannon physics lib in Cables is from 2015. A newer actively maintained fork is developed and called [Cannon ES](https://github.com/pmndrs/cannon-es).

Noticeable features include merging pull requests, bugfixes and speed improvement. Additionally, the webworker [examples](https://github.com/pmndrs/cannon-es/blob/master/examples/worker.html) could really speed up simulations in Cables while being transparent to the user.

**Comment by steam0r** [comment]
hey,

have you tried these? none of their files in dist/ run as a simple browser library but seem to have to be packed and bundled using webpack or something. can you try/find something that runs with customop? thanks!

best,
steam

**Comment by kikohs** [comment]
You can load the new version in a Custom Op using the async module loading.

```
import('https://pmndrs.github.io/cannon-es/dist/cannon-es.js')
  .then((module) => {
    window.MYCANNON = module;
    const w = new window.MYCANNON.World();
    console.log(w);
  });
```

**Comment by steam0r** [comment]
hm, yeah, this is not how cables loads libraries at all...was hoping for something that is loadable with a script-tag, but even their "cjs" thing doesnt load that way.

**Comment by kikohs** [comment]
Yeah I know. I could try to do a browserify version of it if needed.
What bothers me the most is the lack of documentation / examples when you want to use the library.

On 15/09/2021 18:37:00, steam0r ***@***.***> wrote:
hm, yeah, this is not how cables loads libraries at all...was hoping for something that is loadable with a script-tag, but even their "cjs" thing doesnt load that way.

**Comment by steam0r** [comment]
there is nothing that "packs" properly to be included on cables, also no version on any of the javascript cdns, closing this as the alternative would be to build this on our own...
